ChangeLog.d/remove-rsa-mode-parameter.txt

```diff
@@ -0,0 +1,6 @@
+API changes
+   * Remove mode parameter from RSA functions. All encryption,
```

Comment:
There are actually three changes here, and it might make more sense to describe them in separate bullet points. The RSA module no longer supports operations with the wrong mode (and “wrong mode” needs to be explained). That's a removal, which only affects applications that used the wrong mode. Then there's an API change, which is RSA operation functions now have fewer parameters: no more `mode`, and for verify functions, no more RNG. And another API change, which is that the RNG is now mandatory for all private-key RSA operations.